build.sbt

```diff
@@ -88,9 +88,6 @@ lazy val root = (project in file("."))
       commitReleaseVersion,
       tagRelease,
       releaseStepCommandAndRemaining("+publishSigned"),
-      releaseStepCommand("sonatypeBundleRelease"),
-      setNextVersion,
-      commitNextVersion,
-      pushChanges
```

Comment:
I'm assuming we're pulling these out so we can handle the versioning ourselves? I don't have much context around sbt / scala builds, primarily curious.

Reply:
That's correct.